google/cloud/bigquery/v2/minimal/internal/json_utils.cc

```diff
@@ -24,47 +24,89 @@ void FromJson(std::chrono::milliseconds& field, nlohmann::json const& j,
   auto const l = j.find(name);
   if (l == j.end()) return;
   std::int64_t m;
-  l->get_to(m);
-  field = std::chrono::milliseconds(m);
+  if (l->is_string()) {
```

Comment:
This seems like a bit of code to refactor?  Maybe just a function in the anonymous namespace?